internal/backend/db/dbfactory/factory.go

```diff
@@ -20,6 +20,11 @@ var Configs = configset.Set[Config]{
 	Default: &Config{
 		SlowQueryThreshold: 300 * time.Millisecond,
 	},
+	Dev: &Config{
+		Type: "sqlite",
+		DSN:  "file:autokitteh.sqlite",
```

Comment:
1. Shouldn't this be in the default mode as well? After it should be equally or *more* reliable than dev
2. By default we should use the XDG data directory, not the current directory

Reply:
fixed 2

Reply:
1 - now in default mode explicit DSN is required.